src/Nethermind/Nethermind.Blockchain/Receipts/PersistentReceiptStorage.cs

```diff
@@ -237,11 +240,6 @@ public void Insert(Block block, bool updateLowestInserted, params TxReceipt[] tx
                 _transactionDb.Set(txHash, block.Hash.Bytes);
             }
 
-            if (updateLowestInserted && blockNumber < (LowestInsertedReceiptBlock ?? long.MaxValue))
-            {
-                LowestInsertedReceiptBlock = blockNumber;
-            }
-
```

Comment:
Not updating LowestInserted?? Remove it if not needed.

If needed it won't be updated in FullSync!